src/autoformatengine.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Range from '../engine/model/range.js';
+import TextProxy from '../engine/model/textproxy.js';
+
+export default class AutoformatEngine {
```

Comment:
@Reinmar thinks this should be a class, for example, to have `destroy()` capabilities.